### DISCUSSION TITLE
article/7125t7a4/

### DISCUSSION THREAD

**Discussion by giscus** [yxzlwz/blog#7] [Announcements]
# article/7125t7a4/

本人使用谷歌搜索了简中互联网，完全没有找到任何有关 ANAME 的文章……本文该不会是头一份吧 相信大家对于 DNS 的解析方式都不陌生，常见的有 A、CNAME、MX、TXT 记录等等。其中，网站常用的是 A 记录和 CNAME 记录：A 记录用于将域名解析为 IP 地址，CNAME 记录用于将域名解析为另一个域名。而 MX 则是用于邮件服务器的解析...

https://blog.yixiangzhilv.com/blogs/20230714003448.html

**Comment by ReardonChen** [comment]
> 本人使用谷歌搜索了简中互联网，完全没有找到任何有关 ANAME 的文章……本文该不会是头一份吧

大概是因为大部分服务商都是叫做 ALIAS 记录

**Reply by yxzlwz** [reply]
谢谢，学到啦~
